websites/D/dod Games/dist/metadata.json

```
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "nFin1ty",
+		"id": "131877010157862912"
+	},
+	"service": "dod Games",
+	"description": {
+		"en": "dod Games is the most known multiplayer games and chatting community in Greece",
```

This description does not match the description found in the service's metadata

---

Site's description translates to something like "Play live Tichu, Backgammon, Chess, Biriba, UNO, (etc...) on dod.gr Games" if you're referring to that. Thought it was a bit awkard to put that as the game names are not translateable to English and it also looks an awkward advertisement to site, so I created something more descriptive, but I could change that if you prefer the original.

---

@ImRodry What do you think?

---

Sounds good to me as long as you fix it. Also not sure if you wanna say it **is** the most famous, probably one of the most famous
```suggestion
		"en": "dod Games is the most famous multiplayer game and chatting community in Greece",
```